dvc/command/version.py

```diff
@@ -1,30 +1,100 @@
 from __future__ import unicode_literals
 
+import os
+import psutil
 import platform
 import argparse
 import logging
+import uuid
 
+from dvc.utils.compat import Path
+from dvc.repo import Repo
 from dvc.command.base import CmdBaseNoRepo, append_doc_link
 from dvc.version import __version__
+from dvc.exceptions import DvcException, NotDvcRepoError
+from dvc.system import System
 
 
 logger = logging.getLogger(__name__)
 
 
 class CmdVersion(CmdBaseNoRepo):
     def run(self):
+        dvc_version = __version__
+        python_version = platform.python_version()
+        platform_type = platform.platform()
+
         info = (
             "DVC version: {dvc_version}\n"
             "Python version: {python_version}\n"
-            "Platform: {platform_type}"
+            "Platform: {platform_type}\n"
         ).format(
-            dvc_version=__version__,
-            python_version=platform.python_version(),
-            platform_type=platform.platform(),
+            dvc_version=dvc_version,
+            python_version=python_version,
+            platform_type=platform_type,
+        )
+
+        try:
+            repo = Repo()
+            root_directory = repo.find_root()
+
+            info += (
+                "Cache: {cache}\n"
+                "Filesystem type (cache directory): {fs_cache}\n"
+            ).format(
+                cache=self.get_linktype_support_info(repo),
+                fs_cache=self.get_fs_type(repo.cache.local.cache_dir),
+            )
+
+        except NotDvcRepoError:
+            root_directory = os.getcwd()
+
+        info += ("Filesystem type (root directory): {fs_root}").format(
+            fs_root=self.get_fs_type(os.path.abspath(root_directory))
         )
         logger.info(info)
         return 0
 
+    def get_fs_type(self, path):
```

Comment:
Since the method doesn't use `self` and is private:
```suggestion
    @staticmethod
    def _get_fs_type(path):
```